service/src/main/java/edu/tamu/catalog/controller/CatalogAccessController.java

```diff
@@ -64,6 +68,24 @@ public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="e
         }
     }
 
+    /**
+     * Provides data for all fees and fines associated with a patron.
+     *
+     * @param String catalogName (optional)
+     * @param String user
+     * @return
+     */
+    @GetMapping("/fines")
+    public ApiResponse getHolding(@RequestParam(value="catalogName", defaultValue="folio") String catalogName, @WeaverUser User user) {
```

Comment:
Can you rename the method to match the endpoint?

Reply:
Good catch, I think I might also have forget to add `@PreAuthorize("hasRole('USER')")`.
Is this correct?